libs/application/ui-shell/src/components/FormExternalDataProvider.tsx

```diff
@@ -145,6 +153,7 @@ const FormExternalDataProvider: FC<{
   const activateBeforeSubmitCallback = (checked: boolean) => {
     if (checked) {
       setBeforeSubmitCallback(async () => {
+        setSuppressProviderErrors(false)
```

Comment:
Maybe place it after the request so that if the user had errors previously they won't be briefly shown during the fetching of data/